src/System.Windows.Forms/tests/UnitTests/HDPI/ParkingWindowTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using Xunit;
+using static System.Windows.Forms.Application;
+using static Interop;
+
+namespace System.Windows.Forms.Tests.HDPI
+{
+    public partial class ParkingWindowTests : IClassFixture<ThreadExceptionFixture>
+    {
+        [WinFormsFact]
+        public void UnawareParkingWindow()
+        {
+            //set process awareness
+            User32.SetProcessDpiAwarenessContext(User32.DPI_AWARENESS_CONTEXT.PER_MONITOR_AWARE_V2);
+
+            using (DpiHelper.EnterDpiAwarenessScope(User32.DPI_AWARENESS_CONTEXT.UNAWARE))
+            {
+                var control = new Control();
```

Comment:
Here and everywhere else
```suggestion
                using var control = new Control();
```

https://github.com/dotnet/winforms/blob/main/docs/testing.md#dispose-created-objects